tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -767,8 +767,8 @@ TEST_F(ConverterTest, AddAndGetInputs) {
   TF_EXPECT_OK(GetInputs(node_def, &inputs));
 
   EXPECT_EQ(4, inputs.size());
-  EXPECT_EQ(inputs[0].tensor()->simple_tensor(),
-            inputs[1].tensor()->simple_tensor());
+  EXPECT_EQ(inputs[0].tensor()->trt_tensor(),
```

Comment:
These seem unrelated to the PR description.

Reply:
It's not. These hit the assert statement but since the assert was a noop, we didn't see the effect. When we change to CHECK these throw errors and we need to change to "trt_tensor()"

Reply:
@bixia1 this is one place where the tests are incorrect (an assert would be triggered in debug build), but under default profile it doesn't complain. There are a couple other places I fixed as well.

Reply:
SG.